### DISCUSSION TITLE
Selectively revert changes after April 10, 2024

### DISCUSSION THREAD

**Discussion by eradman** [getredash/redash#6964] [General]
In https://github.com/getredash/redash/pull/6876 I proposed a change to revert all commits >= 3f19534301a6b1e2a544c7305238d49e5ba60142 to bring the project back to a stable condition.

Since changes were made to the frontend, server, CI, and build, tests will not pass when rolling back everything. @justinclift asked if we could stage these changes: trying to roll back to a stable condition in a strategic manner seems wise, so my plan of record is to:

1. Revert CI
2. Revert Build
3. Extensively test Rash manually
  a) fix any bugs found
  b) selectively roll back changes to the server and/or UI
4. Reinstate features that were inadvertently removed in steps 1-3

I will post each pull request in this thread.

**Comment by eradman** [comment]
Pull requests for step 1 and 2:
https://github.com/getredash/redash/pull/6965
https://github.com/getredash/redash/pull/6967

**Reply by justinclift** [reply]
#6966 seems to be something else.
#6967 is the one you meant. :smile:

**Reply by eradman** [reply]
yes, thanks

**Comment by eradman** [comment]
Pull request for step 3(b)
https://github.com/getredash/redash/pull/6969

**Comment by eradman** [comment]
Another pull request for step 3(b)
https://github.com/getredash/redash/pull/6971
